_posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.3.0"
+authors:
+- kkondaka
+- dlv
+date: 2023-06-06 10:00:00 -0700
+categories:
+  - releases
+meta_keywords: Data Prepper 2.3.0, ingestion, observability, log pipelines
+meta_description: Data Prepper 2.3.0 with enhancements to expressions, event tagging, enhancements to add_entries processor, s3 sink and tail sampling processor
+---
+
+Data Prepper 2.3.0 is now available for [download](https://opensearch.org/downloads.html#data-prepper)!
+This release introduces a number of changes that help with Data Prepper’s ability to process complex expressions with functions, arithmetic operations, and string operations.
+
+## Enhancements to Data Prepper expressions
+
+Data Prepper 2.3 supports using functions in expressions. A list of supported functions is available in the [Expression syntax](https://opensearch.org/docs/latest/data-prepper/pipelines/expression-syntax/) documentation.
+
+Data Prepper 2.3 supports three types of expressions:
+
+* Conditional expressions
+    * Conditional expressions evaluate to a result of Boolean type, and the expressions now can have functions in them. For example, `length(/message) > 20` would evaluate to true if the length of the message field in the event is greater than 20. Otherwise it evaluates to false. 
```

Comment:
```suggestion
    * Conditional expressions evaluate to a result of Boolean type, and the expressions can now have functions in them. For example, `length(/message) > 20` would evaluate to true if the length of the message field in the event is greater than 20. Otherwise it evaluates to false. 
```